docs/docs/pricing/README.md

```diff
@@ -84,63 +99,42 @@ To discuss an Enterprise plan, please contact the Pipedream Sales Team [using th
 
 Pipedream uses a number of terms to describe platform metrics and details of our plans. See the definitions of key terms below.
 
-### Invocations
+### Credit
 
-Pipedream counts an **invocation** each time a workflow or event source is triggered by an incoming event.
+Pipedream counts your usage in terms of _credits_ of total compute time across all workflows in the billing period.
 
-Pipedream increments the count of invocations by one for each incoming event, regardless of the number of steps in your workflow. For example, if you send an HTTP request to a workflow with five steps, Pipedream will count that as one invocation.
+One credit is {{ $site.themeConfig.base_credits_price.seconds }} seconds of compute time at {{ $site.themeConfig.base_credits_price.memory }} megabytes of memory.
 
-Moreover, if you have a workflow triggered by a cron job running once a minute for the entire day, that will incur 1,440 invocations (60 minute \* 24 hours). For cron-triggered workflows or event sources, remember that you can always reduce the frequency to reduce your invocations.
+Regardless of the number of steps in your workflow, if your workflow run runs for {{ $site.themeConfig.base_credits_price.seconds }} or less seconds then that run only costs 1 credit.
 
-If an event emitted by an event source triggers a single workflow, that will count as **two** invocations: one for the source, and one for the workflow. In other words, source and workflow execution is distinct: each counts invocations on its own.
-
-Your workflow's [memory settings](/workflows/settings/#memory) also impact the number of invocations you're charged for each workflow execution. [Read more here](#how-does-workflow-memory-affect-billable-invocations).
+Adding additional memory capacity to workflows will increase credit usage in intervals of 256 megabytes. For example, doubling the memory of a workflow from {{ $site.themeConfig.base_credits_price.memory }} to {{ $site.themeConfig.base_credits_price.memory * 2 }} will double the cost of credits in the same execution time.
 
```

Comment:
I would add an additional `####` section here for **Source Credit Calculations** and describe the core logic re: sources here:

* First 30 seconds of each dependent, registry source execution is 1 credit. Anything > 30 seconds is charged one credit per 30 second block.
* If you're running an independent source, no source invocations are free. You're charged for all credits according to the standard rules described above.